src/persist-client/Cargo.toml

```diff
@@ -8,19 +8,21 @@ publish = false
 
 [dependencies]
 anyhow = { version = "1.0.56", features = ["backtrace"] }
+async-trait = "0.1.0"
 bincode = "1.3.3"
 bytes = "1.1.0"
+clap = { version = "3.1.8", features = ["derive", "env"] }
```

Comment:
hmmm, this sort of thing is pretty inappropriate to have as a dep on a client library. if we make this whole thing an "example" then we get to put these sort of deps in dev-dependencies

Reply:
I can't quite tell what you'd prefer between this comment and the one below where you're advocating for one binary in persist? or was this comment just musing?

Reply:
not musing :). I think we should have one binary in persist and that binary should be a cargo "example" so that we can use dev-dependencies with it. an alternative is to put it in a separate crate that depends on this one, which I'm not ruling out, but the example is easier to start with